packages/frontend/src/components/DashboardFilter/FilterConfiguration/index.tsx

```diff
@@ -109,12 +109,11 @@ const FilterConfiguration: FC<Props> = ({
 
         if (newField && isField(newField) && isFilterableField(newField)) {
             setDraftFilterRule(
-                createDashboardFilterRuleFromField(
-                    newField,
+                createDashboardFilterRuleFromField({
+                    field: newField,
```

Comment:
this is where it could be _wrong_ but it's not! 
At the moment, if we don't pass a `value` to this functions' object, we set the operator by default to `EQUALS` - this was done before 
ONly if we pass a `value` `null` to it we change its operator to `NULL`